api/v1alpha1/webhooks/release/webhook.go

```diff
@@ -34,6 +37,29 @@ type Webhook struct {
 	log    logr.Logger
 }
 
+func (w *Webhook) Default(ctx context.Context, obj runtime.Object) error {
+	release := obj.(*v1alpha1.Release)
+
+	if release.Spec.GracePeriodDays == 0 {
+		releasePlan := &v1alpha1.ReleasePlan{}
+
+		err := w.client.Get(ctx, types.NamespacedName{
+			Name:      release.Spec.ReleasePlan,
+			Namespace: release.Namespace,
+		}, releasePlan)
+
+		if err != nil && errors.IsNotFound(err) {
+			w.log.Info("releasePlan not found to get ReleaseGracePeriodDays")
```

Comment:
Maybe add like "Not setting release graceperioddays

Reply:
done.

Reply:
* the tests will come in the next commit.